src/transformers/models/layoutlmv2/image_processing_layoutlmv2.py

```diff
@@ -205,7 +205,8 @@ def preprocess(
 
         Args:
             images (`ImageInput`):
-                Image to preprocess.
+                Image to preprocess. Expects a single or batch of images with pixel values ranging from 0 to 255. If
+                passing in images with pixel values between 0 and 1, set `do_rescale=False`.
```

Comment:
`image_processing_layoutlmv2.py` seems not to have a `do_rescale`.  Maybe this doc is not needed here?